app/controllers/admin/volunteers.py

```diff
@@ -102,6 +104,9 @@ def addVolunteer(eventId):
 
     return redirect(url_for('admin.trackVolunteersPage', eventID = eventId))
 
+@admin_bp.route('/addVolunteersToEvent/<username>/<eventId>/isBanned', methods = ['GET'])
+def isVolunteerBanned(username, eventId):
+    return "banned" if isBannedFromEvent(username, eventId) else "not banned"
```

Comment:
I feel that this should be a boolean value to follow the format of our other "is" functions.

Reply:
Fixed